modules/ibmz-configure-devices-manually.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * post-installation-configuration/ibmz-post-install.adoc
+
+:_content-type: PROCEDURE
+[id="configure-additional-devices-manually_{context}"]
+= Configuring additional devices manually 
+
+Tasks in this section describe how to manually configure additional devices in an {ibmzProductName} or LinuxONE environment. This configuration method is persistent over node restarts but not {product-title} native and you need to redo the steps if you replace the node.
+
+.Prerequisites
+
+* You are logged in to the cluster as a user with administrative privileges.
+* The device must be available to the node.
+* In a z/VM environment, the device must be attached to the z/VM guest.
+
+.Procedure
+
+. Connect to the node via SSH:
```

Comment:
Since this is cherry picked I'll just note it here and maybe you can fix it in all of them, but all of our commands need prefaced with "Enter/run the following command:". See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output for more information.